src/components/RegistrationSuccess/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Grow,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import { ReactComponent as FemmecubatorLogo } from './assets/femmecubator-logo.svg';
+import { ReactComponent as CheckCircle } from './assets/check-circle.svg';
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Grow ref={ref} {...props} />;
+});
+
+const useStyles = makeStyles(() => ({
+  dialogWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textAlign: 'center',
+    padding: '2.5rem',
+  },
+  checkCircle: {
+    width: '4rem',
+    height: '4rem',
+    marginTop: '20px',
+  },
+}));
+
+export default function RegistrationSuccess(props) {
+  const { titleText, bodyText, button, openModal } = props;
+  const classes = useStyles();
+
+  return (
+    <Dialog
+      open={openModal}
+      maxWidth="xs"
+      classes={{
+        paper: classes.dialogWrapper,
+      }}
+      TransitionComponent={Transition}
+      disableBackdropClick
+      disableEscapeKeyDown
+      data-testid="registration-success-modal"
```

Comment:
if the attribs are more than 2... use es6 syntax:

`{...{
open: openModal,
      maxWidth: 'xs',
      classes: {
        paper: classes.dialogWrapper,
      },
      TransitionComponent: Transition,
      disableBackdropClick: true,
      disableEscapeKeyDown: true,
      'data-testid': 'registration-success-modal'
}}`